content/blog/2021-05-15-may-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,239 @@
+
+### Curvenote with DVC tutorials
+
+Interested in versioning your data AND your notebooks?  
+[Steve Purves](https://twitter.com/stevejpurves) CTO and co-founder of
+[Curvenote](https://curvenote.com/) gave a three-part tutuorial on integrating
+DVC and Curvenote for creating reproducible, collaborative version control for
+data scientists. The videos are beginner accessbile with tips for intermediate
+git users.
+[Access the videos here.](https://www.youtube.com/watch?v=OnNVbIEIO7A)
+
+![](/uploads/images/2021-05-15/curvenote-dvc-integration.jpeg)_DVC and Curvenote
+for the version control win!_
+
+### CML with Jenkins in DAGsHub
+
+Next up, [Puneeth Pai](https://www.linkedin.com/in/puneeth-pai-b3b299a1/) of
+[Thoughtworks](https://www.thoughtworks.com/) wrote a two-part blog series with
+a how-to for acheiving continuous machine learning using DVC pipelines with
+Jenkins and DAGsHub. Quoted in the article is our own
+[David Ortega](https://github.com/DavidGOrtega),
+
+> Treating experiments like potential new features in a software project opens
+> up many possibilities for improving our engineering practices.
+
+Check out these posts at the link below or catch Puneeth at our next
+[Meetup](https://www.meetup.com/DVC-Community-Virtual-Meetups/events/278163666/)
+where he will be giving a high level overview of this content as well as
+answering questions.
+
+<external-link
+href="https://dagshub.com/blog/in-depth-tour-of-jenkinsfile/"
+title="CML with Jenkins in DAGsHub"
+description="The first of a two-part series on how to set up continuous machine learning using DVC pipelines with Jenkins and DAGsHub."
+link="https://dagshub.com/"
+image="/uploads/images/2021-05-15/puneeth-gears.png"/>
+
+### Discord Server Explosion
+
+Our [Discord server](https://discord.com/invite/dvwXA2N) has exploded since last
+month, up 30% in membership 😱, thanks in large part to a
+[**Towards Data Science**](https://towardsdatascience.com/) post from
+[Sara Metwalli](https://www.linkedin.com/in/sara-a-metwalli/) recommending
+[**9 Discord Servers for Math, Python, and Data Science You Need to Join Today.**](https://towardsdatascience.com/9-discord-servers-for-math-python-and-data-science-you-need-to-join-today-34214b93d6b8)
+
+Sara encourages readers to connect, learn and get inspired. 🚀 Thanks Sara!
+We're on board with that! Rest assured our growing team is hard at work creating
+content, improving tools and working on new tools 😶🤗 to continue to grow and
+serve our MLOps community!
+
+## Next Meetup
+
+Don't miss our
+[Meetup](https://www.meetup.com/DVC-Community-Virtual-Meetups/events/277245660)
+May 27th at 3:00pm UTC, where we will hear from Puneeth Pai as mentioned above
+👆🏽, as well as another user putting DVC and CML into action on his team, and
+finally from David Ortega discussing CML pull requests! Bring your questions!
+We're here to help!
+
+## Tweet Love ❤️
+
+https://twitter.com/FullStackML/status/1389701264997445635?s=20
```

Comment:
Do we have some community Tweets from the last month? :)

Reply:
Inserting Paige's Tweet.  Guess there are pluses to the delay...